Networking/Networking/Model/OrderRefund.swift

```diff
@@ -0,0 +1,112 @@
+import Foundation
+
+
+/// Represents a Order's refund Entity.
+///
+public struct OrderRefund: Decodable {
+    public let siteID: Int
+    public let refundID: Int
+    public let dateCreated: Date
+    public let amount: String
+    public let reason: String
+    public let refunded_by: Int
+    public let items: [OrderItem]
+
+    /// If the refunded_payment field in this response is true, then we can assume that the refund was processed using automatic refund. If false, refund is processed manually.
+    ///
+    public let refunded_payment: Bool
```

Comment:
Same with the name of this property. We tend to name these properties following the lower camel case style: `refundedPayment`

Reply:
Fixed